src/JanusGraph.Net/JanusGraph.Net.csproj

```diff
@@ -21,6 +21,6 @@
     <PackageIconUrl>https://raw.githubusercontent.com/JanusGraph/logos/master/RGB%20-%20PNG%20small/JanusGraph%20logomark%20color%20RGB.png</PackageIconUrl>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Gremlin.Net" Version="3.3.3" />
+    <PackageReference Include="Gremlin.Net" Version="3.4.2" />
```

Comment:
(nitpick) Would it be better to use here `3.4.*` and let the user decide the version?